src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -13,10 +13,8 @@ def bugzilla_url(email_list = '', desc = '')
       cc = ('&cc=' + email_list[1..-1].join('&cc=')) if email_list
     end
 
-    URI.escape(
-      "#{@configuration['bugzilla_url']}/enter_bug.cgi?classification=7340&product=openSUSE.org" \
-      "&component=3rd party software&assigned_to=#{assignee}#{cc}&short_desc=#{desc}"
-    )
+    "#{@configuration['bugzilla_url']}/enter_bug.cgi?classification=7340&product=openSUSE.org" \
+      "&component=3rd%20party%20software&assigned_to=#{CGI.escape(assignee.to_s)}#{CGI.escape(cc.to_s)}&short_desc=#{CGI.escape(desc.to_s)}"
```

Comment:
Consider 

```
{assigned_to: assignee, short_desc: desc, component: '3rd party software'}.to_query
=> "assigned_to=assignee&component=3rd+party+software&short_desc=desc"
```

A little less involved

Reply:
all of this is done in the other PRs now...